Linux build with g++ broken

Appears broken since https://github.com/mapbox/llmr-native/commit/05882c0f54fba654bbbdc40f29b5baf9bb2ddf03.

```
../../src/util/transition.cpp
../../src/util/transition.cpp:31:67: error: specialization of ‘template<class T> llmr::util::transition::state llmr::util::ease_transition<T>::update(llmr::time) const’ in different namespace [-fpermissive]
 transition::state ease_transition<double>::update(llmr::time now) const {
                                                                   ^
In file included from ../../src/util/transition.cpp:1:0:
../../include/llmr/util/transition.hpp:51:11: error:   from definition of ‘template<class T> llmr::util::transition::state llmr::util::ease_transition<T>::update(llmr::time) const’ [-fpermissive]
     state update(time now) const;
           ^
../../src/util/transition.cpp:43:72: error: specialization of ‘template<class T> llmr::util::transition::state llmr::util::ease_transition<T>::update(llmr::time) const’ in different namespace [-fpermissive]
 transition::state ease_transition<llmr::Color>::update(llmr::time now) const {
                                                                        ^
In file included from ../../src/util/transition.cpp:1:0:
../../include/llmr/util/transition.hpp:51:11: error:   from definition of ‘template<class T> llmr::util::transition::state llmr::util::ease_transition<T>::update(llmr::time) const’ [-fpermissive]
     state update(time now) const;
           ^
../../src/util/transition.cpp:55:66: error: specialization of ‘template<class T> llmr::util::transition::state llmr::util::ease_transition<T>::update(llmr::time) const’ in different namespace [-fpermissive]
 transition::state ease_transition<float>::update(llmr::time now) const {
                                                                  ^
In file included from ../../src/util/transition.cpp:1:0:
../../include/llmr/util/transition.hpp:51:11: error:   from definition of ‘template<class T> llmr::util::transition::state llmr::util::ease_transition<T>::update(llmr::time) const’ [-fpermissive]
     state update(time now) const;
           ^
../../src/util/transition.cpp:67:81: error: specialization of ‘template<class T> llmr::util::transition::state llmr::util::ease_transition<T>::update(llmr::time) const’ in different namespace [-fpermissive]
 transition::state ease_transition<std::array<float, 2>>::update(llmr::time now) const {
                                                                                 ^
In file included from ../../src/util/transition.cpp:1:0:
../../include/llmr/util/transition.hpp:51:11: error:   from definition of ‘template<class T> llmr::util::transition::state llmr::util::ease_transition<T>::update(llmr::time) const’ [-fpermissive]
     state update(time now) const;
           ^
make[1]: *** [../Release/obj.target/llmr-x86/src/util/transition.o] Error 1
make[1]: Leaving directory `/home/travis/build/mapbox/llmr-native/build/llmr'
make: *** [llmr] Error 2

The command "make llmr" exited with 2.
```

--- a/src/util/transition.cpp
+++ b/src/util/transition.cpp
@@ -2,7 +2,7 @@
 #include <llmr/util/unitbezier.hpp>
 #include <llmr/platform/platform.hpp>
 
-using namespace llmr::util;
+namespace llmr { namespace util {
 
 UnitBezier ease(0.25, 0.1, 0.25, 1);
 
@@ -74,3 +74,5 @@ transition::state ease_transition<std::array<float, 2>>::update(llmr::time now) 
         return running;
     }
 }
+
+}}

